eng/pipelines/templates/stages/archetype-ios-release.yml

```diff
@@ -66,17 +66,32 @@ stages:
 
           - deployment: PublishPackageToSwiftPackageManagerMirrors
             displayName: "Publish to SPM Mirrors"
-            condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
+            condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'), ne('${{artifact.skipSwiftPackageManager}}', true))
             environment: cocoapodstrunk
             dependsOn: TagRepository
             variables:
               - template: ../variables/globals.yml
             pool:
-              name: Azure Pipelines
-              vmImage: $(OSVmImage)
+              name: azsdk-pool-mms-ubuntu-1804-general
+              vmImage: MMSUbuntu18.04
             strategy:
               runOnce:
                 deploy:
                   steps:
-                    - script: |
-                        echo "Pretend we are publishing to Swift Package Manager mirrors"
+                    - checkout: self
+                    - checkout: SwiftPM-${{artifact.name}}
+                      persistCredentials: true
+                    - pwsh: |
+                        $podspec = Get-Content -Raw -Path $(Pipeline.Workspace)/s/azure-sdk-for-ios/sdk/${{parameters.ServiceDirectory}}/${{artifact.name}}/${{artifact.name}}.podspec.json | ConvertFrom-Json
```

Comment:
There are a lot of steps here which might be difficult to debug if something goes wrong. We should add some logging and perhaps refactoring some of this into a script.

Reply:
I was debating a script on this one, I think you are right, I'll extract it into a script.

Reply:
Its now in a script.